### ISSUE TITLE
Rename of URL to something else

### ISSUE THREAD

**Issue by niftylettuce** [unshiftio/url-parse#145]
Can we please rename the `URL` function?  ESLint compat is throwing errors since it's a global defined already.  The code is here <https://github.com/unshiftio/url-parse/blob/master/index.js#L149>.  This variable name conflicts with the URL API https://caniuse.com/#feat=url, and ESLint compat doesn't know to check for a function in scope with same name.

```sh
  3419:24  error  URL is not supported in Opera Mobile 12.1, Opera Mini all, IE Mobile 10, IE 9, Blackberry Browser 7                compat/compat
  3421:24  error  URL is not supported in Opera Mobile 12.1, Opera Mini all, IE Mobile 10, IE 9, Blackberry Browser 7                compat/compat
  3509:12  error  URL is not supported in Opera Mobile 12.1, Opera Mini all, IE Mobile 10, IE 9, Blackberry Browser 7                compat/compat
```

**Comment by lpinca** [comment]
I'm fine with renaming it to `Url`.

**Comment by 3rd-Eden** [comment]
Fix released in 1.4.2
